lib/views/help/privacy.html.erb

```diff
@@ -2,573 +2,895 @@
+      <td>30 minutes</td>
+    </tr>
+    <tr>
+      <td>__utmz</td>
+      <td>Information on how the site was reached (e.g. direct or via a
+      link/search/advertisement)</td><td>6 months</td>
+    </tr>
+    <tr>
+      <td>__utmx</td><td>Which variation of a page you are seeing if we are
+      testing different versions to see which is best</td><td>2 years</td>
+    </tr>
+  </table>
+  <h4>Google’s Official Statement about Analytics Data</h4>
+  <p>
+    “This website uses Google Analytics, a web analytics service provided by
+    Google, Inc. (“Google”).  Google Analytics uses “cookies”, which are
+    text files placed on your computer, to help the website analyze how
+    users use the site. The information generated by the cookie about your
+    use of the website (including your IP address) will be transmitted to
+    and stored by Google on servers in the United States . Google will use
+    this information for the purpose of evaluating your use of the website,
+    compiling reports on website activity for website operators and
+    providing other services relating to website activity and internet
+    usage.  Google may also transfer this information to third parties where
+    required to do so by law, or where such third parties process the
+    information on Google’s behalf. Google will not associate your IP
+    address with any other data held by Google.  You may refuse the use of
+    cookies by selecting the appropriate settings on your browser, however
+    please note that if you do this you may not be able to use the full
+    functionality of this website.  By using this website, you consent to
+    the processing of data about you by Google in the manner and for the
+    purposes set out above.”
+  </p>
+  <p><strong>Measuring website performance (New Relic)</strong></p>
+  <p>
+    WhatDoTheyKnow uses New Relic to collect data on the performance of the
+    site - how quickly it sends pages and how much memory and processing power
+    it takes on the computers that run it. We do this to ensure that the site
+    runs quickly and efficiently. New Relic stores information on:
+  </p>
+  <ul>
+    <li>
+      What pages are visited on the site, and how long they take to load.
+    </li>
+    <li>
+      The structure, not the content, of database queries made in order to
+      run the site and how long they take to run.
+    </li>
+    <li>
+      Memory and CPU usage on the servers that run the site.
+    </li>
+  </ul>
+  <p>
+    Sample data about database queries, individual requests for pages is
+    stored by New Relic for 7 days, aggregate data is stored for a maximum
+    of 90 days.
+  </p>
+  <p>
+    <a href="https://newrelic.com/security">
+    Further information from New Relic on privacy and security</a>
+  </p>
+  <p>
+    <a href="https://www.mysociety.org/privacy/">More general information on
+    how third party services work</a>
+  </p>
+  <p><strong>Our own logging</strong></p>
```

Comment:
This section already exists (with slightly different wording) at `/privacy#our_logging`